docs/sample_inventories/rbac-over-oauth-and-mtls.yml

```diff
@@ -70,10 +70,15 @@ all:
     # configs to talk to mtls only kafka Listener.
     schema_registry_kafka_listener_name: puremtls
     kafka_connect_kafka_listener_name: puremtls
-    kafka_rest_kafka_listener_name: puremtls
     ksql_kafka_listener_name: puremtls
     control_center_kafka_listener_name: puremtls
 
+    # In above we didn't set kafka_rest listener name to puremtls as this results in elevated access.
```

Comment:
i'm not sure if these should be documented from ansible , please confirm with the Cp security team

Reply:
Updated the comment after discussing with CP security team